frontends/web/src/components/guide/guide.tsx

```diff
@@ -37,13 +37,19 @@ export const store = new Store<SharedProps>({
     guideExists: false,
 });
 
-apiGet('config').then(({ frontend }) => {
-    if (frontend && frontend.guideShown != null) { // eslint-disable-line eqeqeq
-        store.setState({ shown: frontend.guideShown });
-    } else {
-        store.setState({ shown: true });
-    }
-});
+// if apiGet() is invoked imediatelly this can error due to cyclinc dependencies
```

Comment:
is this a new problem or was this always buggy?

typos: imediatelly -> immediately, cyclinc -> cyclic

Reply:
As far as I know data fetching should only be done within React class lifecycle methods (useEffect, componentDidMount) which would have circumvented this issue, but I can't find any documentation on that.